crates/editor/src/mouse_context_menu.rs

```diff
@@ -90,7 +91,12 @@ pub fn deploy_context_menu(
                 .action("Copy", Box::new(Copy))
                 .action("Paste", Box::new(Paste))
                 .separator()
-                .action("Reveal in Finder", Box::new(RevealInFinder))
+                .when(cfg!(target_os = "macos"), |builder| {
+                    builder.action("Reveal in Finder", Box::new(RevealInFileManager))
+                })
+                .when(cfg!(not(target_os = "macos")), |builder| {
+                    builder.action("Reveal in File Manager", Box::new(RevealInFileManager))
+                })
```

Comment:
How about compilation check?
```suggestion
                action(ACTION_NAME, Box::new(RevealInFileManager))
```

And somewhere:
```rust
#[cfg(target_os = "macos")]
const ACTION_NAME: &str = "Reveal in Finder";
```

Reply:
This is just my personal thought :) . Personally, I prefer compiling checks over runtime checks. The final decision is yours.

Reply:
@JunkuiZhang I am implementing your suggestion. I just have one question. I don't know all the source code well yet, but I guess the action name can be inserted in some crate or some specific place to be then used throughout the source code. Any suggestions?

For what I've seen now in the source code I could put it in `editor/src/actions.rs`, does that work for you? Or would you prefer that I put it in the same file where it is used for the moment?

Reply:
Looks good to me :)

Reply:
I think I disagree with this personally, the original version says what's going on very explicitly. That said, neither implementation would be a blocker :)

Reply:
So in this case we can consider this issue/PL closed? @mikayla-maki